conf/defaults.ini

```diff
@@ -779,6 +779,12 @@ skip_org_role_sync = false
 sync_cron = "0 1 * * *"
 active_sync_enabled = true
 
+############# OAuth2 Server for external service/plugin auth #############
```

Comment:
Should we have these while we are behind a feature toggle?
Maybe we could add a comment to state that it's alpha?

Reply:
Yeah, it's better to remove it from the `defaults.ini`, because the config parameters of the Extended JWT client had been removed as well.

(I have already pushed the change.)